graders/python/python_autograder/pl_execute.py

```diff
@@ -81,7 +91,7 @@ def execute_code(
             str_student = extract_ipynb_contents(f, ipynb_key)
         else:
             str_student = f.read()
-    str_student = str_leading + "\n" + str_student + "\n" + str_trailing
+    str_student = "\n".join(filter(bool, [str_leading, str_student, str_trailing]))
```

Comment:
```suggestion
    str_student = "\n".join(filter(bool, (str_leading, str_student, str_trailing)))
```
Nit: Slight preference for tuple because it's immutable. Cool use of the filter function!

Reply:
Changed in 52d3263!